### ISSUE TITLE
activate.py injects non-relocatable path into conda.sh

### ISSUE THREAD

**Issue by dbast** [conda/conda#12182]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

activate.py injects non-relocatable path into conda.sh during conda-build with cygwin-bash while building conda with `python setup.py install`, leading to a broken conda.sh as reported in https://github.com/conda/conda/issues/12176

Running `python -c "import os, json; print(json.dumps(dict(os.environ), sort_keys=True, indent=4))"` through our internal build system shows what variables contain a PATH with `/cygdrive/c/b` instead of `C:\\b\\`, see below.

So only the variables `CYGWIN_PREFIX` and `PWD` contain a non-relocatable `/cygdrive/c/` prefixed path. Both variables are not exported/provided by [conda-build](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#id2) and thus should not be used in `activate.py` to compose + inject variables into conda.sh upon install.

**Comment by dbast** [comment]
<details>
  <summary>Click me</summary>

  ```js
           INFO   {                                                                                                                                                                                                                                                                                                                                             
           INFO       "ALLUSERSPROFILE": "C:\\ProgramData",                                                                                                                                                                                                                                                                                                     
           INFO       "APPDATA": "C:\\Users\\builder-admin\\AppData\\Roaming",                                                                                                                                                                                                                                                                                  
           INFO       "APR": "1.6.3",                                                                                                                                                                                                                                                                                                                           
           INFO       "ARCH": "64",                                                                                                                                                                                                                                                                                                                             
           INFO       "AWS_EXECUTION_ENV": "EC2",                                                                                                                                                                                                                                                                                                               
           INFO       "BLAS_IMPL": "mkl",                                                                                                                                                                                                                                                                                                                       
           INFO       "BOOST": "1.73",                                                                                                                                                                                                                                                                                                                          
           INFO       "BOOST_CPP": "1.73",                                                                                                                                                                                                                                                                                                                      
           INFO       "BUILD": "amd64-pc-windows-19.0.0",                                                                                                                                                                                                                                                                                                       
           INFO       "BUILD_PLATFORM": "win-64",                                                                                                                                                                                                                                                                                                               
           INFO       "BUILD_PREFIX": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_build_env",                                                                                                                                                                                                                                                        
           INFO       "BZIP2": "1.0",                                                                                                                                                                                                                                                                                                                           
           INFO       "CAIRO": "1.16",                                                                                                                                                                                                                                                                                                                          
           INFO       "CHANNEL_TARGETS": "defaults",                                                                                                                                                                                                                                                                                                            
           INFO       "CHOCOLATEYINSTALL": "C:\\ProgramData\\chocolatey",                                                                                                                                                                                                                                                                                       
           INFO       "CHOCOLATEYLASTPATHUPDATE": "133154315156757239",                                                                                                                                                                                                                                                                                         
           INFO       "CLANG_VARIANT": "clang",                                                                                                                                                                                                                                                                                                                 
           INFO       "CMAKE_GENERATOR": "Visual Studio 14 2015 Win64",                                                                                                                                                                                                                                                                                         
           INFO       "COMMONPROGRAMFILES": "C:\\Program Files\\Common Files",                                                                                                                                                                                                                                                                                  
           INFO       "COMMONPROGRAMFILES(X86)": "C:\\Program Files (x86)\\Common Files",                                                                                                                                                                                                                                                                       
           INFO       "COMMONPROGRAMW6432": "C:\\Program Files\\Common Files",                                                                                                                                                                                                                                                                                  
           INFO       "COMPUTERNAME": "EC2AMAZ-U8C7G3B",                                                                                                                                                                                                                                                                                                        
           INFO       "COMSPEC": "C:\\Windows\\system32\\cmd.exe",                                                                                                                                                                                                                                                                                              
           INFO       "CONDA_ALLOW_SOFTLINKS": "false",                                                                                                                                                                                                                                                                                                         
           INFO       "CONDA_BUILD": "1",                                                                                                                                                                                                                                                                                                                       
           INFO       "CONDA_BUILD_STATE": "BUILD",                                                                                                                                                                                                                                                                                                             
           INFO       "CONDA_DEFAULT_ENV": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env",                                                                                                                                                                                                                                                       
           INFO       "CONDA_EXE": "C:\\miniconda3\\condabin\\..\\Scripts\\conda.exe",                                                                                                                                                                                                                                                                          
           INFO       "CONDA_LUA": "5",                                                                                                                                                                                                                                                                                                                         
           INFO       "CONDA_NPY": "1.21",                                                                                                                                                                                                                                                                                                                      
           INFO       "CONDA_PERL": "5.26",                                                                                                                                                                                                                                                                                                                     
           INFO       "CONDA_PKGS_DIRS": "C:\\b\\abs_95_x7r8z1c\\pkg_dirs",                                                                                                                                                                                                                                                                                     
           INFO       "CONDA_PREFIX": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env",                                                                                                                                                                                                                                                            
           INFO       "CONDA_PREFIX_1": "C:\\miniconda3",                                                                                                                                                                                                                                                                                                       
           INFO       "CONDA_PROMPT_MODIFIER": "(C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env) ",                                                                                                                                                                                                                                                
           INFO       "CONDA_PY": "310",                                                                                                                                                                                                                                                                                                                        
           INFO       "CONDA_PYTHON_EXE": "C:\\miniconda3\\python.exe",                                                                                                                                                                                                                                                                                         
           INFO       "CONDA_R": "3.4",                                                                                                                                                                                                                                                                                                                         
           INFO       "CONDA_ROOT": "C:\\miniconda3",                                                                                                                                                                                                                                                                                                           
           INFO       "CONDA_SHLVL": "2",                                                                                                                                                                                                                                                                                                                       
           INFO       "CPU_COUNT": "4",                                                                                                                                                                                                                                                                                                                         
           INFO       "CPU_OPTIMIZATION_TARGET": "nocona",                                                                                                                                                                                                                                                                                                      
           INFO       "CRAN_MIRROR": "https://mran.microsoft.com/snapshot/2018-01-01",                                                                                                                                                                                                                                                                          
           INFO       "CROSS_COMPILER_TARGET_PLATFORM": "win-64",                                                                                                                                                                                                                                                                                               
           INFO       "CXX_COMPILER": "vs2017",                                                                                                                                                                                                                                                                                                                 
           INFO       "CYGWIN_PREFIX": "/cygdrive/c/b/abs_95_x7r8z1c/croot/conda_1671090233545/_h_env",                                                                                                                                                                                                                                                       
           INFO       "CYRUS_SASL": "2.1.26",                                                                                                                                                                                                                                                                                                                   
           INFO       "C_COMPILER": "vs2017",                                                                                                                                                                                                                                                                                                                   
           INFO       "DASK_PARENT": "3884",                                                                                                                                                                                                                                                                                                                    
           INFO       "DBUS": "1",                                                                                                                                                                                                                                                                                                                              
           INFO       "DISTUTILS_USE_SDK": "1",                                                                                                                                                                                                                                                                                                                 
12/15/22 02:44:49  INFO       "DRIVERDATA": "C:\\Windows\\System32\\Drivers\\DriverData",                                                                                                                                                                                                                                                                               
           INFO       "EC2LAUNCH_TELEMETRY": "1",                                                                                                                                                                                                                                                                                                               
           INFO       "EXPAT": "2",                                                                                                                                                                                                                                                                                                                             
           INFO       "EXTEND_KEYS": "{'extend_keys', 'ignore_version', 'pin_run_as_build', 'ignore_build_only_deps'}",                                                                                                                                                                                                                                         
           INFO       "FONTCONFIG": "2.14",                                                                                                                                                                                                                                                                                                                     
           INFO       "FORTRAN_COMPILER": "intel-fortran",                                                                                                                                                                                                                                                                                                      
           INFO       "FORTRAN_COMPILER_VERSION": "2022.1.0",                                                                                                                                                                                                                                                                                                   
           INFO       "FREETYPE": "2.10",                                                                                                                                                                                                                                                                                                                       
           INFO       "G2CLIB": "1.6",                                                                                                                                                                                                                                                                                                                          
           INFO       "GEOS": "3.8.0",                                                                                                                                                                                                                                                                                                                          
           INFO       "GIFLIB": "5",                                                                                                                                                                                                                                                                                                                            
           INFO       "GLIB": "2",                                                                                                                                                                                                                                                                                                                              
           INFO       "GMP": "6.1",                                                                                                                                                                                                                                                                                                                             
           INFO       "GNU": "2.12.2",                                                                                                                                                                                                                                                                                                                          
           INFO       "GSTREAMER": "1.18",                                                                                                                                                                                                                                                                                                                      
           INFO       "GST_PLUGINS_BASE": "1.18",                                                                                                                                                                                                                                                                                                               
           INFO       "HARFBUZZ": "4.3.0",                                                                                                                                                                                                                                                                                                                      
           INFO       "HDF4": "4.2",                                                                                                                                                                                                                                                                                                                            
           INFO       "HDF5": "1.10.6",                                                                                                                                                                                                                                                                                                                         
           INFO       "HDFEOS2": "2.20",                                                                                                                                                                                                                                                                                                                        
           INFO       "HDFEOS5": "5.1",                                                                                                                                                                                                                                                                                                                         
           INFO       "HOME": "C:\\cygwin64\\home\\builder-admin",                                                                                                                                                                                                                                                                                              
           INFO       "HOSTNAME": "EC2AMAZ-U8C7G3B",                                                                                                                                                                                                                                                                                                            
           INFO       "ICU": "58",                                                                                                                                                                                                                                                                                                                              
           INFO       "IFORT_COMPILER22": "C:\\Program Files (x86)\\Intel\\oneAPI\\compiler\\2022.1.0\\windows\\",                                                                                                                                                                                                                                              
           INFO       "IGNORE_BUILD_ONLY_DEPS": "{'numpy', 'python'}",                                                                                                                                                                                                                                                                                          
           INFO       "INCLUDE": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\include;",                                                                                                                                                                                                                                              
           INFO       "INFOPATH": "/usr/local/info:/usr/share/info:/usr/info",                                                                                                                                                                                                                                                                                  
           INFO       "INSPECTOR_2022_DIR": "C:\\Program Files (x86)\\Intel\\oneAPI\\inspector\\2022.1.0",                                                                                                                                                                                                                                                      
           INFO       "I_MPI_ONEAPI_ROOT": "C:\\Program Files (x86)\\Intel\\oneAPI\\mpi\\2021.6.0",                                                                                                                                                                                                                                                             
           INFO       "JPEG": "9",                                                                                                                                                                                                                                                                                                                              
           INFO       "LANG": "en_US.UTF-8",                                                                                                                                                                                                                                                                                                                    
           INFO       "LIB": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\lib;",                                                                                                                                                                                                                                                      
           INFO       "LIBDAP4": "3.19",                                                                                                                                                                                                                                                                                                                        
           INFO       "LIBFFI": "3.4",                                                                                                                                                                                                                                                                                                                          
           INFO       "LIBGD": "2.3.3",                                                                                                                                                                                                                                                                                                                         
           INFO       "LIBGDAL": "3.0",                                                                                                                                                                                                                                                                                                                         
           INFO       "LIBGSASL": "1.8",                                                                                                                                                                                                                                                                                                                        
           INFO       "LIBKML": "1.3",                                                                                                                                                                                                                                                                                                                          
           INFO       "LIBNETCDF": "4.8",                                                                                                                                                                                                                                                                                                                       
           INFO       "LIBPNG": "1.6",                                                                                                                                                                                                                                                                                                                          
           INFO       "LIBPROTOBUF": "3.20.1",                                                                                                                                                                                                                                                                                                                  
           INFO       "LIBRARY_BIN": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\bin",                                                                                                                                                                                                                                               
           INFO       "LIBRARY_INC": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\include",                                                                                                                                                                                                                                           
           INFO       "LIBRARY_LIB": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\lib",                                                                                                                                                                                                                                               
           INFO       "LIBRARY_PREFIX": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library",                                                                                                                                                                                                                                                 
           INFO       "LIBTIFF": "4.1",                                                                                                                                                                                                                                                                                                                         
           INFO       "LIBWEBP": "1.2.0",                                                                                                                                                                                                                                                                                                                       
           INFO       "LIBXML2": "2.9",                                                                                                                                                                                                                                                                                                                         
           INFO       "LIBXSLT": "1.1",                                                                                                                                                                                                                                                                                                                         
           INFO       "LLVM_VARIANT": "llvm",                                                                                                                                                                                                                                                                                                                   
           INFO       "LOCALAPPDATA": "C:\\Users\\builder-admin\\AppData\\Local",                                                                                                                                                                                                                                                                               
           INFO       "LUA_VER": "5",                                                                                                                                                                                                                                                                                                                           
           INFO       "LZO": "2",                                                                                                                                                                                                                                                                                                                               
           INFO       "M2W64_CXX_COMPILER": "m2w64-toolchain",                                                                                                                                                                                                                                                                                                  
           INFO       "M2W64_C_COMPILER": "m2w64-toolchain",                                                                                                                                                                                                                                                                                                    
           INFO       "M2W64_FORTRAN_COMPILER": "m2w64-toolchain",                                                                                                                                                                                                                                                                                              
           INFO       "MALLOC_TRIM_THRESHOLD_": "65536",                                                                                                                                                                                                                                                                                                        
           INFO       "MKL": "2021.*",                                                                                                                                                                                                                                                                                                                          
           INFO       "MKL_NUM_THREADS": "1",                                                                                                                                                                                                                                                                                                                   
12/15/22 02:44:50  INFO       "MPFR": "4",                                                                                                                                                                                                                                                                                                                              
           INFO       "MSSDK": "1",                                                                                                                                                                                                                                                                                                                             
           INFO       "MSYS2_ARG_CONV_EXCL": "/AI;/AL;/OUT;/out",                                                                                                                                                                                                                                                                                               
           INFO       "MSYS2_ENV_CONV_EXCL": "CL",                                                                                                                                                                                                                                                                                                              
           INFO       "NPY_DISTUTILS_APPEND_FLAGS": "1",                                                                                                                                                                                                                                                                                                        
           INFO       "NPY_VER": "1.21",                                                                                                                                                                                                                                                                                                                        
           INFO       "NUMBER_OF_PROCESSORS": "4",                                                                                                                                                                                                                                                                                                              
           INFO       "OLDPWD": "/home/builder-admin",                                                                                                                                                                                                                                                                                                          
           INFO       "OMP_NUM_THREADS": "1",                                                                                                                                                                                                                                                                                                                   
           INFO       "ONEAPI_ROOT": "C:\\Program Files (x86)\\Intel\\oneAPI\\",                                                                                                                                                                                                                                                                                
           INFO       "OPENBLAS": "0.3.20",                                                                                                                                                                                                                                                                                                                     
           INFO       "OPENJPEG": "2.3",                                                                                                                                                                                                                                                                                                                        
           INFO       "OPENSSL": "1.1.1",                                                                                                                                                                                                                                                                                                                       
           INFO       "ORIGINAL_PATH": "/cygdrive/c/miniconda3:/cygdrive/c/miniconda3/Library/mingw-w64/bin:/cygdrive/c/miniconda3/Library/usr/bin:/cygdrive/c/miniconda3/Library/bin:/cygdrive/c/miniconda3/Scripts:/cygdrive/c/miniconda3/bin:/cygdrive/c/miniconda3/condabin:/cygdrive/c/Program Files                                                       
                  (x86)/Intel/oneAPI/tbb/latest/redist/intel64/vc_mt:/cygdrive/c/Program Files (x86)/Intel/oneAPI/tbb/latest/redist/ia32/vc_mt:/cygdrive/c/Program Files (x86)/Intel/oneAPI/mpi/latest/bin:/cygdrive/c/Program Files (x86)/Intel/oneAPI/mpi/latest/bin/release:/cygdrive/c/Program Files                                                        
                  (x86)/Intel/oneAPI/mpi/latest/libfabric/bin:/cygdrive/c/Program Files (x86)/Intel/oneAPI/mpi/latest/libfabric/bin/utils:/cygdrive/c/Program Files (x86)/Intel/oneAPI/compiler/latest/windows/redist/intel64_win/compiler:/cygdrive/c/Program Files                                                                                            
                  (x86)/Intel/oneAPI/compiler/latest/windows/redist/ia32_win/compiler:/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/Wbem:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0:/cygdrive/c/Windows/System32/OpenSSH:/cygdrive/c/Program                                                                           
                  Files/Amazon/cfn-bootstrap:/cygdrive/c/ProgramData/chocolatey/bin:/cygdrive/c/Program Files/Amazon/AWSCLIV2:/cygdrive/c/Program Files/OpenSSH-Win64:/cygdrive/c/Users/builder-admin/AppData/Local/Microsoft/WindowsApps",                                                                                                                     
           INFO       "OS": "Windows_NT",                                                                                                                                                                                                                                                                                                                       
           INFO       "PATH":                                                                                                                                                                                                                                                                                                                                   
                  "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_build_env;C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_build_env\\Library\\mingw-w64\\bin;C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_build_env\\Library\\usr\\bin;C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_build_env\\Library\\bin;C:\\b\\abs_95_x7r8…
                  Files (x86)\\Intel\\oneAPI\\tbb\\latest\\redist\\intel64\\vc_mt;C:\\Program Files (x86)\\Intel\\oneAPI\\tbb\\latest\\redist\\ia32\\vc_mt;C:\\Program Files (x86)\\Intel\\oneAPI\\mpi\\latest\\bin;C:\\Program Files (x86)\\Intel\\oneAPI\\mpi\\latest\\bin\\release;C:\\Program Files                                                         
                  (x86)\\Intel\\oneAPI\\mpi\\latest\\libfabric\\bin;C:\\Program Files (x86)\\Intel\\oneAPI\\mpi\\latest\\libfabric\\bin\\utils;C:\\Program Files (x86)\\Intel\\oneAPI\\compiler\\latest\\windows\\redist\\intel64_win\\compiler;C:\\Program Files                                                                                               
                  (x86)\\Intel\\oneAPI\\compiler\\latest\\windows\\redist\\ia32_win\\compiler;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0;C:\\Windows\\System32\\OpenSSH;C:\\Program Files\\Amazon\\cfn-bootstrap;C:\\ProgramData\\chocolatey\\bin;C:\\Program                                 
                  Files\\Amazon\\AWSCLIV2;C:\\Program Files\\OpenSSH-Win64;C:\\Users\\builder-admin\\AppData\\Local\\Microsoft\\WindowsApps",                                                                                                                                                                                                                   
           INFO       "PATHEXT": ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC",                                                                                                                                                                                                                                                                       
           INFO       "PERL_VER": "5.26",                                                                                                                                                                                                                                                                                                                       
           INFO       "PIN_RUN_AS_BUILD": "OrderedDict([('python', OrderedDict([('min_pin', 'x.x'), ('max_pin', 'x.x')])), ('r-base', OrderedDict([('min_pin', 'x.x'), ('max_pin', 'x.x')])), ('libboost', {'max_pin': 'x.x.x'})])",                                                                                                                            
           INFO       "PIP_CACHE_DIR": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\pip_cache",                                                                                                                                                                                                                                                        
           INFO       "PIP_IGNORE_INSTALLED": "True",                                                                                                                                                                                                                                                                                                           
           INFO       "PIP_NO_BUILD_ISOLATION": "False",                                                                                                                                                                                                                                                                                                        
           INFO       "PIP_NO_DEPENDENCIES": "True",                                                                                                                                                                                                                                                                                                            
           INFO       "PIP_NO_INDEX": "True",                                                                                                                                                                                                                                                                                                                   
           INFO       "PIXMAN": "0.40",                                                                                                                                                                                                                                                                                                                         
           INFO       "PKG_BUILDNUM": "3",                                                                                                                                                                                                                                                                                                                      
           INFO       "PKG_BUILD_STRING": "placeholder",                                                                                                                                                                                                                                                                                                        
           INFO       "PKG_HASH": "1234567",                                                                                                                                                                                                                                                                                                                    
           INFO       "PKG_NAME": "conda",                                                                                                                                                                                                                                                                                                                    
           INFO       "PKG_VERSION": "1.4",                                                                                                                                                                                                                                                                                                                     
           INFO       "PREFIX": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env",                                                                                                                                                                                                                                                                  
           INFO       "PROCESSOR_ARCHITECTURE": "AMD64",                                                                                                                                                                                                                                                                                                        
           INFO       "PROCESSOR_IDENTIFIER": "Intel64 Family 6 Model 85 Stepping 4, GenuineIntel",                                                                                                                                                                                                                                                             
           INFO       "PROCESSOR_LEVEL": "6",                                                                                                                                                                                                                                                                                                                   
           INFO       "PROCESSOR_REVISION": "5504",                                                                                                                                                                                                                                                                                                             
           INFO       "PROFILEREAD": "true",                                                                                                                                                                                                                                                                                                                    
           INFO       "PROGRAMDATA": "C:\\ProgramData",                                                                                                                                                                                                                                                                                                         
           INFO       "PROGRAMFILES": "C:\\Program Files",                                                                                                                                                                                                                                                                                                      
           INFO       "PROGRAMFILES(X86)": "C:\\Program Files (x86)",                                                                                                                                                                                                                                                                                           
           INFO       "PROGRAMW6432": "C:\\Program Files",                                                                                                                                                                                                                                                                                                      
           INFO       "PROJ": "6.2.1",                                                                                                                                                                                                                                                                                                                          
           INFO       "PROJ4": "5.2.0",                                                                                                                                                                                                                                                                                                                         
           INFO       "PROMPT": "(C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env) $P$G",                                                                                                                                                                                                                                                           
           INFO       "PSMODULEPATH": "C:\\Program Files\\WindowsPowerShell\\Modules;C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files (x86)\\AWS Tools\\PowerShell\\",                                                                                                                                                                 
           INFO       "PUBLIC": "C:\\Users\\Public",                                                                                                                                                                                                                                                                                                            
           INFO       "PWD": "/cygdrive/c/b/abs_95_x7r8z1c/clone/.abs",                                                                                                                                                                                                                                                                                         
           INFO       "PY3K": "1",                                                                                                                                                                                                                                                                                                                              
           INFO       "PYTHON": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\python.exe",                                                                                                                                                                                                                                                      
           INFO       "PYTHONNOUSERSITE": "1",                                                                                                                                                                                                                                                                                                                  
           INFO       "PYTHON_IMPL": "cpython",                                                                                                                                                                                                                                                                                                                 
           INFO       "PYTHON_IMPLEMENTATION": "cpython",                                                                                                                                                                                                                                                                                                       
           INFO       "PY_VCRUNTIME_REDIST": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Library\\bin\\vcruntime140.dll",                                                                                                                                                                                                                     
           INFO       "PY_VER": "3.10",                                                                                                                                                                                                                                                                                                                         
           INFO       "READLINE": "8.0",                                                                                                                                                                                                                                                                                                                        
           INFO       "RECIPE_DIR": "C:\\b\\abs_95_x7r8z1c\\clone\\recipe",                                                                                                                                                                                                                                                                                     
           INFO       "ROOT": "C:\\miniconda3",                                                                                                                                                                                                                                                                                                                 
           INFO       "RUST_COMPILER": "rust-gnu",                                                                                                                                                                                                                                                                                                              
           INFO       "RUST_COMPILER_VERSION": "1.56.0",                                                                                                                                                                                                                                                                                                        
           INFO       "R_BASE": "3.4",                                                                                                                                                                                                                                                                                                                          
           INFO       "R_IMPLEMENTATION": "r-base",                                                                                                                                                                                                                                                                                                             
           INFO       "R_VER": "3.4",                                                                                                                                                                                                                                                                                                                           
           INFO       "R_VERSION": "3.5.0",                                                                                                                                                                                                                                                                                                                     
12/15/22 02:44:51  INFO       "SCRIPTS": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Scripts",                                                                                                                                                                                                                                                        
           INFO       "SERF": "1.3.9",                                                                                                                                                                                                                                                                                                                          
           INFO       "SHELL": "/bin/bash",                                                                                                                                                                                                                                                                                                                     
           INFO       "SHLIB_EXT": ".dll",                                                                                                                                                                                                                                                                                                                      
           INFO       "SHLVL": "1",                                                                                                                                                                                                                                                                                                                             
           INFO       "SP_DIR": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Lib\\site-packages",                                                                                                                                                                                                                                              
           INFO       "SQLITE": "3",                                                                                                                                                                                                                                                                                                                            
           INFO       "SRC_DIR": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\work",                                                                                                                                                                                                                                                                   
           INFO       "STDLIB_DIR": "C:\\b\\abs_95_x7r8z1c\\croot\\conda_1671090233545\\_h_env\\Lib",                                                                                                                                                                                                                                                         
           INFO       "SUBDIR": "win-64",                                                                                                                                                                                                                                                                                                                       
           INFO       "SYSTEMDRIVE": "C:",                                                                                                                                                                                                                                                                                                                      
           INFO       "SYSTEMROOT": "C:\\Windows",                                                                                                                                                                                                                                                                                                              
           INFO       "SYS_PREFIX": "C:\\miniconda3",                                                                                                                                                                                                                                                                                                           
           INFO       "SYS_PYTHON": "C:\\miniconda3\\python.exe",                                                                                                                                                                                                                                                                                               
           INFO       "TARGET_PLATFORM": "win-64",                                                                                                                                                                                                                                                                                                              
           INFO       "TEMP": "C:\\cygwin64\\tmp",                                                                                                                                                                                                                                                                                                              
           INFO       "TERM": "xterm-256color",                                                                                                                                                                                                                                                                                                                 
           INFO       "TK": "8.6",                                                                                                                                                                                                                                                                                                                              
           INFO       "TMP": "C:\\cygwin64\\tmp",                                                                                                                                                                                                                                                                                                               
           INFO       "TZ": "Etc/UTC",                                                                                                                                                                                                                                                                                                                          
           INFO       "USER": "builder-admin",                                                                                                                                                                                                                                                                                                                  
           INFO       "USERDOMAIN": "EC2AMAZ-U8C7G3B",                                                                                                                                                                                                                                                                                                          
           INFO       "USERNAME": "builder-admin",                                                                                                                                                                                                                                                                                                              
           INFO       "USERPROFILE": "C:\\Users\\builder-admin",                                                                                                                                                                                                                                                                                                
           INFO       "VC": "14",                                                                                                                                                                                                                                                                                                                               
           INFO       "VERBOSE_AT": "V=1",                                                                                                                                                                                                                                                                                                                      
           INFO       "VERBOSE_CM": "VERBOSE=1",                                                                                                                                                                                                                                                                                                                
           INFO       "VS2017INSTALLDIR": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools",                                                                                                                                                                                                                                                 
           INFO       "VS2019INSTALLDIR": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools",                                                                                                                                                                                                                                                 
           INFO       "VS_MAJOR": "14",                                                                                                                                                                                                                                                                                                                         
           INFO       "VS_VERSION": "14.0",                                                                                                                                                                                                                                                                                                                     
           INFO       "VS_YEAR": "2015",                                                                                                                                                                                                                                                                                                                        
           INFO       "WINDIR": "C:\\Windows",                                                                                                                                                                                                                                                                                                                  
           INFO       "XZ": "5",                                                                                                                                                                                                                                                                                                                                
           INFO       "ZIP_KEYS": "[['python', 'numpy']]",                                                                                                                                                                                                                                                                                                      
           INFO       "ZLIB": "1.2",                                                                                                                                                                                                                                                                                                                            
           INFO       "ZSTD": "1.5.2",                                                                                                                                                                                                                                                                                                                          
           INFO       "_": "/cygdrive/c/miniconda3/Scripts/conda"                                                                                                                                                                                                                                                                                               
           INFO   }   
  ```
</details>

**Comment by dbast** [comment]
Maybe this can be fixed by updating the package recipe (instead of changes to the activate.py), by changing https://github.com/AnacondaRecipes/conda-feedstock/blob/master/recipe/bld.bat and e.g. the sed https://github.com/AnacondaRecipes/conda-feedstock/blob/master/recipe/build.sh#L10

**Comment by kenodegard** [comment]
@dbast is this the same issue as https://github.com/conda/conda/issues/12626 and resolved by the recent PR https://github.com/conda/conda/pull/12627?

**Comment by kenodegard** [comment]
after reviewing https://github.com/conda/conda/issues/12176 this does indeed appear to be a duplciate of https://github.com/conda/conda/issues/12626 and was resolved in conda 23.5.0
